build.sbt

```diff
@@ -125,9 +125,9 @@ lazy val coreS3 = project
     name := s"$baseName-s3",
     libraryDependencies ++= Seq(
       "com.lightbend.akka" %% "akka-stream-alpakka-s3" % alpakkaVersion,
+      "com.typesafe.akka"  %% "akka-http-xml"          % akkaHttpVersion,
```

Comment:
Can you add the comment here?
> I had to move the various akka-http-* dependencies from Test to normal compile scope because an older version of akka-http was being resolved in the dependency scanning which happened to have a security vulne

Reply:
Pushed a commit with an explanation